Add news post for playtest-20180102.

pchote opened the pull request:
TODO:
- [x] Final changelog link
- [x] Battlefield news
- [x] Playtest release ID

![20180101-fullpage](https://user-images.githubusercontent.com/167819/34462269-ca476786-ee37-11e7-8a86-421cb04ccd0a.png)

penev92 commented on content/news/playtest-20180101.md:
```diff
@@ -0,0 +1,64 @@
+
+OpenRA playtest builds do not overwrite the release version if you are on Windows or macOS, and you can switch between the two when joining an online server. So head over to our [download page](/download/) and give it a try!
+
+The last development cycle has brought some significant improvements for modders, but we recommend that projects currently based on the stable release to hold off upgrading for now. Experienced OpenRA modders well know the difficulties of updating your code for a new OpenRA release, and this playtest includes more changes than most. We are planning major improvements to our semi-automatic mod-upg ...
```
Need to fix the "recommend that someone to do something" construct.

reaperrr replied:
In that first sentence I'd remove the 'to' before and add 'from' after the 'hold off'.

reaperrr commented on content/news/playtest-20180101.md:
```diff
@@ -0,0 +1,64 @@
+We are excited to welcome the new year with a new [OpenRA playtest](/downloads/)!
+During the last 5 months we merged over 700 commits from 42 developers, with each commit working towards fixing bugs, improving code quality, or adding new features that improve the single- and multi-player experience. 
```
Maybe 'contributors' instead of 'developers'?

abcdefg30 commented on content/news/playtest-20180101.md:
```diff
@@ -0,0 +1,63 @@
+
+OpenRA playtest builds do not overwrite the release version if you are on Windows or macOS, and you can switch between the two when joining an online server. So head over to our [download page](/download/) and give it a try!
+
+This development cycle brought significant improvements for community mods, but we recommend that modders wait for a future playtest before updating your projects.  We plan to merge a new version of our semi-automated rule upgrader before the final release, which will siimplify the process of updating your [Mod SDK](https://github.com/OpenRA/OpenRAModSDK) based projects to the latest engine versi ...
```
`siimplify`

reaperrr replied:
Also, first sentence: In my opinion 'their projects' would be more consistent with 'modders'.

abcdefg30 commented on content/news/playtest-20180101.md:
```diff
@@ -0,0 +1,63 @@
+Other notable changes include:
+
+* New lobby options to disable bounty, build radius, and re-packing Construction Yards
+* Units will no longer target units revealed by the Red Alert GPS Satellite
```
What about replacing the second `units` by `enemies` (so we don't have units twice in the same sentence)?

abcdefg30 commented on content/news/playtest-20180101.md:
```diff
@@ -0,0 +1,63 @@
+
+ * The full Harkonnen campaign in D2K
+ * Refinements and improvements to several campaign missions in Red Alert and Tiberian Dawn
+ * A fix for the AI cheating when it targets superweapons and support powers
```
`targets` might be confusing here. At first I thought the AI is targeting the powers. We could use `fires` instead.

abcdefg30 commented on content/news/playtest-20180101.md:
```diff
@@ -0,0 +1,63 @@
+
+OpenRA playtest builds do not overwrite the release version if you are on Windows or macOS, and you can switch between the two when joining an online server. So head over to our [download page](/download/) and give it a try!
+
+This development cycle brought significant improvements for community mods, but we recommend that modders wait for a future playtest before updating your projects.  We plan to merge a new version of our semi-automated rule upgrader before the final release, which will siimplify the process of updating your [Mod SDK](https://github.com/OpenRA/OpenRAModSDK) based projects to the latest engine versi ...
```
"that modders wait [...] before updating your projects." 3rd person versus 2nd person? your -> their?

abcdefg30 commented on content/news/playtest-20180101.md:
```diff
@@ -0,0 +1,63 @@
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180101-serverplayers.png" width="600" alt="Server player list">
+
+The new server player list is one of several improvements to the multiplayer UI
```
Missing full stop.

abcdefg30 commented on content/news/playtest-20180101.md:
```diff
@@ -0,0 +1,63 @@
+
+This playtest is a timely opportunity to get community participation, as we celebrate a massive record of games played. At the end of November we passed 500,000 completed multiplayer games since August 2014, when we started recording online server statistics. This is a huge accomplishment for our community, congratulations!
+
+The main focus of this development cycle was modernizing core parts of the engine and game code to support the work-in-progress Tiberian Sun mod and other community projects. These shouldn't have visibly changed our released mods, but big changes always have a risk of introducing unexpected bugs or side-effects. Please note that changes to the way hotkeys are managed means you will need to redefi ...
```
`changes means` versus `changes mean`?

pchote commented:
Updated and added battlefield news:

![screen shot 2018-01-01 at 14 13 43](https://user-images.githubusercontent.com/167819/34468327-3669804a-eefe-11e7-8626-489d4c6a2cbb.png)

penev92 commented on content/news/playtest-20180101.md:
```diff
@@ -0,0 +1,63 @@
+Platform integration fixes include "Pin to taskbar" support on Windows and 64 bit support for macOS.
+</div>
+
+As always, we suggest reading the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog) for more information and to view the full list of changes.
```
"reading and to view" sounds meh

pchote commented:
Updated.

penev92 commented:
LGTM 👍
